pravega/pravega-benchmark
# Issue 25: Remove com.emc reference in package names

Change log description
com.emc.pravega references in package names are replaced with io.pravega.

Purpose of the change
Fixes #25 .

What the code does
the directory structure com.emc.pravega is changed as io.pravega and package names in all the src file are changed as io.pravega

How to verify it
Run ./gradlew build should generate the  ./build/distributions/pravega-benchmark.tar file. untaring this file yields the executable ./build/distributions/pravega-benchmark/bin/pravega-benchmark . This executable should work performing the bench-marking of pravega cluster.

Signed-off-by: Keshava Munegowda keshava.munegowda@dell.com
